include/edyn/math/matrix3x3.hpp

```diff
@@ -159,7 +159,7 @@ inline matrix3x3 inverse_matrix_symmetric(const matrix3x3 &m) {
     auto a22 = m[1][1], a23 = m[1][2];
     auto a33 = m[2][2];
 
-    matrix3x3 m_inv;
+    matrix3x3 m_inv{};
```

Comment:
Isn't it ok to leave this uninitialized?

Reply:
It's not constexpr otherwise, no clue why exactly.
Clang yells at me if i don't initialize it.